Akkadu/rsi-widgets
# Feature/multilangual

Please describe the purpose of the pull request.

* The purpose was to implement our new multilingual feature on the `interpretation-player`

## Motivation and Context



## How To Test



* On akkadu.com
* Event name : MULTI-LANGUAGE-2
* Interpreter accounts:
1- French : romain@akkadu-team.com
`akkadu_team_1234`

2- Chinese : abelbordonado@gmail.com
`abel123456`
@abelbordonado  I'm using your account:)


* On the Vue or React repo you should be able to 
  * 1- When there is no **relay language** (the floor is the original floor (en-US))
    * When you switch to `English, the audio of the video's audio should be unmuted.
    * When you switch to `French-Chinese, you should be able to listen to your interpretation. And the audio of the video should be muted.  
    
   * 2- When there is a relay language. Example `french`. On the french interpreter account, you are outputting `English
    * When switching to `English you should now listen to your `french` interpretation and the video should be muted.
    * When switching to `french` you should now listen to the video's audio.



## Screenshots (if appropriate)

## Checklist



加油！
